### ISSUE TITLE
Overfitting problem

### ISSUE THREAD

**Issue by 87nohigher** [HHTseng/video-classification#53]
All three model have overfitting issue
Even the best model 2D Resnet152-CNN + LSTM
the training accuracy and validation accuracy have 15% gap.
Is there any way to overcome the overfitting problem?
